src/components/HighlightedPosts/index.module.scss

```diff
@@ -0,0 +1,112 @@
+@import "common/breakpoints";
+
+.root {
+  position: relative;
+
+  margin-bottom: 89px;
```

Comment:
I believe best practice is to move layout spacings to the parent / a layout component, etc.

Reply:
Normally yes, and I was the person that came up with that rule in our context xD but I think we can safely ignore it because its purpose is to make components re-usable, and this one is 100% not going to be re-used

Reply:
Sure. Though you're also removing the predicability of where spacings are set.

Reply:
You think so? I can never guess where the margins are until I do "inspect element" and look at which component it is. I'm not against following the rules; it's just that the rules are not supposed to be applied blindly.